doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -49,14 +49,15 @@ This chapter is organized as follows:
     - VIM Core services (keystone, cinder, nova, neutron etc.)
     - Tenant Separation
     - Host aggregates providing resource pooling
-    - Flavor management
+    - Flavor* management
 *	Underlying Resources: are what provides the resources that allow the Consumable Infrastructure Resources and Services to be created and managed by the Cloud Infrastructure Management Software (VIM).
     - Virtualisation
     - Physical infrastructure
       -	Compute
       -	Network: Spine/Leaf; East/West and North/South traffic
       -	Storage
 
+> * Please note "flavours" is used in the Reference Model and shall continue to be used in the context of specify the geometry of the virtual resources. The term "flavor" will be used in OpenStack context including when specifying configurations; the OpenSTack term flavor includes what profile configuration information as "extra specs".
```

Comment:
Typo: OpenSTack
Is "what" necessary in the last sentence?

Reply:
@karinesevilla Corrected that and other errors - Thanks